helm/cluster-eks/values.yaml

```diff
@@ -97,7 +97,7 @@ global:
       audit: true
       authenticator: true
       controllerManager: true
-    oidcIdentityProviderConfig: {}
+    oidc: {}
```

Comment:
whilst the field in the ManagedControlPlane is still `oidcIdentityProviderConfig`, we use `.global.controlPlane.oidc` in the `cluster` chart so I've aligned the values schema with it. The create managed control plane resource values are unchanged